nautobot/core/testing/filters.py

```diff
@@ -41,7 +41,7 @@ def get_filterset_test_values(self, field_name, queryset=None):
                 # randomly break out of loop after 2 values have been selected
                 if len(test_values) > 1 and random.choice([True, False]):  # noqa: S311  # suspicious-non-cryptographic-random-usage
                     break
-                if value[field_name] and value["count"] < qs_count:
+                if value[field_name] and (len(test_values) < 2 or value["count"] < qs_count):
```

Comment:
I think this makes it so that we always pick the first two values that we find, rather than random ones - is that change intentional/necessary?

Reply:
revert it for now see what breaks